gangsta_chess/logic/board/chess_board.py

```diff
@@ -13,11 +13,23 @@ class ChessBoard(Board):
     def __init__(self, Pieces, shape=(8, 8)):
         super().__init__(Pieces, shape)
         self.back_row = np.array([Rook, Knight, Bishop, King, Queen, Bishop, Knight, Rook])
+        self._place_pieces()
 
-    def place_pieces(self):
+    def _place_pieces(self):
```

Comment:
I kinda like this being public and calling it from outside the constructor so the constructor doesn't do too much but it's up to you because it's nothing huge.